### ISSUE TITLE
Issue setting bucket (and potentially object) tags

### ISSUE THREAD

**Issue by Ajnbro** [minio/minio-py#1052]
I am using minio-py 7.0.0 and receive a MalformedXML S3Error any time I try to set the tags of my bucket. This issue probably occurs with object tags as well, but I have not tested that yet.

The simplified code below follows your API example code but seems to fail. `bucket_name` and `s3Client` client are defined elsewhere in my code and everything is properly imported within my code.

```
tags = Tags.new_bucket_tags()
tags["Project"] = "Project One"
tags["User"] = "jsmith"

try:
    s3Client.set_bucket_tags(bucket_name, tags)
except (ValueError, S3Error) as exc:
    print(str(exc))
```

Here is the error I am receiving:
```S3 operation failed; code: MalformedXML, message: The XML you provided was not well-formed or did not validate against our published schema, resource: None, request_id: ****, host_id: ****```

**Comment by balamurugana** [comment]
@Ajnbro Can you provide HTTP traces by enabling `client.trace_on(sys.stdout)`?

**Comment by Ajnbro** [comment]
@balamurugana 
```
---------START-HTTP---------
GET /testeitrbucket?location= HTTP/1.1
Host: s3.us-east-1.amazonaws.com
User-Agent: MinIO (Linux; aarch64) minio-py/7.0.0
X-Amz-Content-Sha256: UNSIGNED-PAYLOAD
X-Amz-Date: 20210104T182239Z
Authorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20210104/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*

HTTP/1.1 200
x-amz-id-2: TG5pYts+e/M94eYe5iJcMsYdmjCSvtbB8Z4Iu1quQWL8SY2OMgNqEYb5kAIO1p6q4uyAYPeQN9I=
x-amz-request-id: 9DB3D2857900A2BA
Date: Mon, 04 Jan 2021 18:22:40 GMT
Content-Type: application/xml
Transfer-Encoding: chunked
Server: AmazonS3
----------END-HTTP----------
---------START-HTTP---------
PUT /?tagging= HTTP/1.1
Content-Md5: z0crmX+B6F7at6ncrMGzDQ==
Host: testeitrbucket.s3.us-east-1.amazonaws.com
User-Agent: MinIO (Linux; aarch64) minio-py/7.0.0
Content-Length: 119
X-Amz-Content-Sha256: UNSIGNED-PAYLOAD
X-Amz-Date: 20210104T182239Z
Authorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20210104/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*

HTTP/1.1 400
x-amz-request-id: 288C821A2B77DE19
x-amz-id-2: 0+JTAIgiY+aUaeXf9OiMV/eseWksdoUY9vATRBgfKZT4BBQCRkXSNOWAVZnGBFdsYi8qqw/SDqc=
Content-Type: application/xml
Transfer-Encoding: chunked
Date: Mon, 04 Jan 2021 18:22:39 GMT
Connection: close
Server: AmazonS3
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>MalformedXML</Code><Message>The XML you provided was not well-formed or did not validate against our published schema</Message><RequestId>288C821A2B77DE19</RequestId><HostId>0+JTAIgiY+aUaeXf9OiMV/eseWksdoUY9vATRBgfKZT4BBQCRkXSNOWAVZnGBFdsYi8qqw/SDqc=</HostId></Error>
----------END-HTTP----------
S3 operation failed; code: MalformedXML, message: The XML you provided was not well-formed or did not validate against our published schema, resource: None, request_id: 288C821A2B77DE19, host_id: 0+JTAIgiY+aUaeXf9OiMV/eseWksdoUY9vATRBgfKZT4BBQCRkXSNOWAVZnGBFdsYi8qqw/SDqc=
```
